assets/prefabs/CookingRecipes.prefab

```diff
@@ -248,6 +248,90 @@
         },
         "outputCount": 1
       }
+      "Cooking:BoiledEgg": {
+        "inputs": {
+          "Cooking:Egg": 1
+        },
+        "outputCount": 1
+      }
+      "Cooking:CookedBean": {
+        "inputs": {
+          "Cooking:bean":1,
+          "Cooking:onion":1,
+          "Cooking:tomato":1,
```

Comment:
According to https://github.com/Terasology/Cooking/blob/develop/assets/prefabs/recipe/cook/RecipeCookedBean.prefab#L27, this needs to be 3 input tomatoes.